### ISSUE TITLE
Examples for using code (not from command line?)

### ISSUE THREAD

**Issue by ddofer** [songlab-cal/tape#65]
Hi - is there an option to provide some examples on how to use the models (pretrained or novel) to train/fine-tune on new tasks, or the existing tasks, not just from the command line? 

(The models and classes (e.g. tokenizer) are not 1:1 with huggingface's format, so it makes applying it to new problems hard. Even a simple example of training on a given set of sequences and labels [in code] would be great - currently i'm running into issues applying these models, or running my own on them (e.g. loading the data etc').

**Comment by rmrao** [comment]
I'm not sure what this would be other than to look at the training code and data loading code that already exists? You can look at the `run_train` function ([here](https://github.com/songlab-cal/tape/blob/ab49d242a81f7a1d0b291507c8a0b0f5fd6d960c/tape/training.py#L386)) and walk through what it's doing. I'd suggest adding some pdb statements and seeing where data, etc. are loaded.

I don't really have the bandwidth to write a completely separate training script, and all my existing training code goes through `run_train` since it's built to be highly generalizable.

**Comment by rmrao** [comment]
Also, have you looked in the examples folder?
